### PR TITLE
Release v0.25.0

### PR DESCRIPTION
**Pull request by dashpole**
https://github.com/google/cadvisor/issues/1566

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+### 0.25.0 (2017-03-09)
+- disable thin_ls due to excessive iops
+- ignore .mount cgroups, fixing dissappearing stats
+- Fix wc goroutine leak
+- use sha256 for cAdvisor releases
```
Comment (timstclair) [inline CHANGELOG.md]: you can drop this

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+### 0.25.0 (2017-03-09)
+- disable thin_ls due to excessive iops
```
Comment (timstclair) [inline CHANGELOG.md]: nit: consistent capitalization. start each line with uppercase

**Comment by dashpole** [comment]
@k8s-bot test this

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+- ignore .mount cgroups, fixing dissappearing stats
+- Fix wc goroutine leak
+- use sha256 for cAdvisor releases
+- update aws-sdk-go dependency to 1.6.10
+- cAdvisor tested with latest 1.7 go release
```
Comment (timstclair) [inline CHANGELOG.md]: I'd reword this to "Update to go 1.7 for releases"

File: CHANGELOG.md (hunk 2 above)
Comment (dashpole) [inline CHANGELOG.md]: done

File: CHANGELOG.md (hunk 1 above)
Comment (dashpole) [inline CHANGELOG.md]: done.

File: CHANGELOG.md (hunk 3 above)
Comment (dashpole) [inline CHANGELOG.md]: done.
